server/app/serializers/fusor/subscription_serializer.rb

```diff
@@ -2,7 +2,9 @@ module Fusor
   class SubscriptionSerializer < ActiveModel::Serializer
 
     embed :ids, include: true
-    attributes :id, :deployment_id, :contract_number, :product_name, :quantity_attached, :start_date, :end_date, :total_quantity, :source
+    attributes :id, :deployment_id, :contract_number, :product_name,
+               :quantity_attached, :start_date, :end_date,
+               :total_quantity, :source, :quantity_to_add
```

Comment:
@jmrodri, the culprit was that I didn't have an updated version of `subscription_serializer` on my desktop which didn't have `quantity_to_add`. Checkout a fresh branch again from the PR and hopefully it will work.